lib/Dialect/Comb/CombOps.cpp

```diff
@@ -157,6 +158,18 @@ static unsigned getTotalWidth(ValueRange inputs) {
   return resultWidth;
 }
 
+static LogicalResult verifyConcatOp(ConcatOp concatOp) {
+  unsigned tyWidth = concatOp.getType().getWidth();
+  unsigned operandsTotalWidth = getTotalWidth(concatOp.inputs());
+  if (tyWidth != operandsTotalWidth)
+    return concatOp.emitOpError(llvm::formatv(
```

Comment:
Please use stream insertion instead of llvm::format for the emitError things.  You should be able to do things like:

`return concatOp.emitOpError("some stuff") << aThing << "some stuff" << another thing;`